mypy/build.py

```diff
@@ -883,8 +936,24 @@ def find() -> Optional[str]:
                     find_module_isdir_cache[pathitem, dir_chain] = isdir
                 if isdir:
                     dirs.append(dir)
-            find_module_dir_cache[dir_chain, lib_path] = dirs
-        candidate_base_dirs = find_module_dir_cache[dir_chain, lib_path]
+
+        # Third-party stub/typed packages
+        for pkg_dir in package_dirs:
+            stub_name = components[0] + '_stubs'
```

Comment:
Shouldn't this be `-stubs` now?

(Also is there no test code for this?)

Reply:
Yes I didn't get to update the implementation until now. Also the tests are at https://github.com/ethanhs/mypy/blob/d74eae824da4bf1cbd530f5a0586e4d2985850da/mypy/test/testpackages.py#L47 and blow with the nested with statements (which checks resolution is correct). Perhaps the name of the test method can be better.